### PR TITLE
Bundle update all the tings

### PR DESCRIPTION
**Pull request by chrishanes**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -286,4 +292,7 @@ Lint/UselessAccessModifier:
 Lint/IneffectiveAccessModifier:
   Enabled: false
 
+Lint/AmbiguousBlockAssociation:
+  Enabled: false
+
```
Comment (alan) [inline .rubocop.yml]: Have you looked into these new cops? Are any of them worth considering enabling in the future?

File: app/helpers/form_elements_helper.rb [hunk 2]
```diff
@@ -85,16 +85,18 @@ def label_wrapped_single_field(form, name, field_method, *options)
...
-        form.label(name, class: label_class) {
-          join(
-            form.public_send(field_method, name, *options),
-            t(".#{name}"),
-            field_hint(name),
-            field_error(form, name)
-          )
-        }
-      )
+      content_tag(:div, class: 'multiple-choice'){
```
Comment (alan) [inline app/helpers/form_elements_helper.rb]: space between `){`

File: app/models/prison.rb [hunk 3]
```diff
@@ -18,9 +18,9 @@ class Prison < ActiveRecord::Base
     to: :parsed_slot_details
   delegate :finder_slug, :nomis_id, to: :estate
 
-  scope :enabled, lambda {
+  scope :enabled, (lambda {
```
Comment (alan) [inline app/models/prison.rb]: Would the stabby lambda work better here?

File: app/models/visit.rb [hunk 4]
```diff
@@ -31,29 +31,29 @@ class Visit < ActiveRecord::Base
   delegate :allowance_will_renew?, :allowance_renews_on,
     to: :rejection
 
-  scope :from_estates, lambda { |estates|
+  scope :from_estates, (lambda { |estates|
```
Comment (alan) [inline app/models/visit.rb]: Are all these changes because of rubocop?

File: app/views/prison/visits/_cancel_date.html.erb [hunk 5]
```diff
@@ -5,22 +5,28 @@
       <div class="grid-row">
         <fieldset>
           <div class="column-one-third">
-            <label for="cancellation_reason_booked_in_error" class="block-label">
-              <%= radio_button_tag 'cancellation_reason', :booked_in_error, false %>
-              <%= t(".booked_in_error") %>
-            </label>
+            <div class="multiple-choice">
+            <%= radio_button_tag 'cancellation_reason', :booked_in_error, false %>
```
Comment (alan) [inline app/views/prison/visits/_cancel_date.html.erb]: Indentation of these 3 `radio_button_tag`'s is off

File: app/views/prison/visits/_cancel_visitor.html.erb [hunk 6]
```diff
@@ -5,10 +5,12 @@
       <div class="grid-row">
         <fieldset>
           <div class="column-one-third">
-            <label for="cancellation_reason_visitor_banned" class="block-label">
-              <%= radio_button_tag 'cancellation_reason', :visitor_banned, false %>
-              <%= t(".visitor_banned") %>
-            </label>
+            <div class="multiple-choice">
+            <%= radio_button_tag 'cancellation_reason', :visitor_banned, false %>
```
Comment (alan) [inline app/views/prison/visits/_cancel_visitor.html.erb]: Indentation

File: config/environments/development.rb [hunk 7]
```diff
@@ -1,3 +1,5 @@
+require 'uglifier'
```
Comment (alan) [inline config/environments/development.rb]: Is this needed?

File: spec/features/process_a_request_reject_spec.rb [hunk 8]
```diff
@@ -19,7 +19,7 @@
     end
 
     scenario 'rejecting a booking with no available slot' do
-      choose 'None of the chosen times are available'
+      choose 'None of the chosen times are available', visible: false
```
Comment (alan) [inline spec/features/process_a_request_reject_spec.rb]: How come all these inputs are now not visible?

File: spec/metrics/weekly_metrics_confirmed_csv_exporter_spec.rb [hunk 9]
```diff
@@ -27,11 +27,11 @@
       week1 = week_ago.beginning_of_week.to_date.to_s
       week2 = two_weeks_ago.beginning_of_week.to_date.to_s
 
-      expect(subject).to eq(<<-CSV)
-Prison,#{week1},#{week2}
-A Prison,1,0
-B Prison,0,1
-C Prison,0,0
+      expect(subject).to eq(<<~CSV)
```
Comment (alan) [inline spec/metrics/weekly_metrics_confirmed_csv_exporter_spec.rb]: Nice, didn't know about `<<~`

File: .rubocop.yml (hunk 1 above)
Comment (StupidCodeFactory) [inline .rubocop.yml]: they make rspec ```change { something }``` throw warnings. don't think it's a massive deal.

**Comment by alan** [comment]
@StupidCodeFactory did you run the integration tests against this branch? Did you see my comments before merging this?

**Pull request merged**